### PR TITLE
Update Events Pages “hacktoberfest-2020/index”

### PR DESCRIPTION
**Pull request by isaac-pulumi**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/events/hacktoberfest-2020/index.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
-meta_desc: Join Pulumi for Hacktoberfest and help us build open source integrations, automation and libraries.
+meta_desc: Join Pulumi for Hacktoberfest and help us build open source
+  integrations, automation and libraries.
 title: Hacktoberfest 2020
 subtitle: Help us build new open source capabilities for Pulumi
 unlisted: false
+redirect_to: https://organize.mlh.io/participants/events/4954-pulumi-hacktoberfest
 event:
-  registration_url: https://hacktoberfest.digitalocean.com
+  eventbrite_url: " "
```
Comment (isaac-pulumi) [inline content/events/hacktoberfest-2020/index.md:9]:
```suggestion
  eventbrite_url: ""
```

File: content/events/hacktoberfest-2020/index.md [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
+  registration_url: https://organize.mlh.io/participants/events/4954-pulumi-hacktoberfest
+  calendly_url: " "
```
Comment (isaac-pulumi) [inline content/events/hacktoberfest-2020/index.md:11]:
```suggestion
  calendly_url: ""
```

File: content/events/hacktoberfest-2020/index.md [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
+  eventbrite_id: " "
```
Comment (isaac-pulumi) [inline content/events/hacktoberfest-2020/index.md:12]:
```suggestion
  eventbrite_id: ""
```

File: content/events/hacktoberfest-2020/index.md [hunk 4]
```diff
@@ -1,10 +1,15 @@
 ---
-meta_desc: Join Pulumi for Hacktoberfest and help us build open source integrations, automation and libraries.
+meta_desc: Join Pulumi for Hacktoberfest and help us build open source
+  integrations, automation and libraries.
 title: Hacktoberfest 2020
 subtitle: Help us build new open source capabilities for Pulumi
 unlisted: false
+redirect_to: https://organize.mlh.io/participants/events/4954-pulumi-hacktoberfest
```
Comment (isaac-pulumi) [inline content/events/hacktoberfest-2020/index.md:7]:
```suggestion
```

Comment (isaac-pulumi) [inline content/events/hacktoberfest-2020/index.md]:
```suggestion
```

**Review by isaac-pulumi — COMMENTED** [review]
removing 'redirect'

**Review by zchase — APPROVED** [review]
LGTM

**Pull request merged**